pallets/automation-time/src/lib.rs

```diff
@@ -769,5 +781,14 @@ pub mod pallet {
 				TaskHashInput::<T> { owner_id: owner_id.clone(), provided_id: provided_id.clone() };
 			T::Hashing::hash_of(&task_hash_input)
 		}
+
+		fn calculate_execution_fee(action: &Action<T>) -> BalanceOf<T> {
+			let raw_weight = match action {
+				Action::Notify { message: _ } => 1_000u32,
+				Action::NativeTransfer { sender: _, recipient: _, amount: _ } => 2_000u32,
+			};
+			let weight = <BalanceOf<T>>::from(raw_weight);
+			T::ExecutionWeightFee::get().saturating_mul(weight)
```

Comment:
why is execution weight fee static? shouldn't it scale with size?

Reply:
can someone explain the resolution to this pls?